backend/src/test/java/gov/cdc/usds/simplereport/service/PatientBulkUploadServiceTest.java

```diff
@@ -108,8 +108,8 @@ void withoutFacilityId_validCsvUploadsToOrganization() {
     // WHEN
     this._service.processPersonCSV(inputStream, null);
 
-    assertThat(getPatientsForFacility(firstFacilityId).size())
-        .isEqualTo(getPatientsForFacility(secondFacilityId).size());
+    assertThat(getPatientsForFacility(firstFacilityId))
+        .hasSameClassAs(getPatientsForFacility(secondFacilityId));
```

Comment:
Won't `hasSameClassAs` just assert that they are both just `List<Person>`? I don't think this asserts that they are equal though.